src/routes/v2/address.ts

```diff
@@ -141,70 +136,75 @@ async function detailsBulk(
     }
 
     // Enforce no more than 20 addresses.
-    if (addresses.length > 20) {
-      res.json({
+    if (addresses.length > FREEMIUM_INPUT_SIZE) {
+      res.status(400)
+      return res.json({
```

Comment:
Express doesn't care if `return` is used before `res.json()`, but the unit tests mocks of Express *do* depend on `return` being used.